pages/api/get-phone-number-from-ticket.ts

```diff
@@ -0,0 +1,30 @@
+import { NextApiRequest, NextApiResponse } from "next"
+import { withSentry } from "@sentry/nextjs"
+import { directusApiInstance as directusApiInstanceInitiator, TicketService } from "../../src/services/directus-api"
+
+let directusApiInstance: TicketService
+const getDirectusApiInstance = (): TicketService => {
+  if (!directusApiInstance) {
+    directusApiInstance = new TicketService(directusApiInstanceInitiator())
+  }
+  return directusApiInstance
+}
```

Comment:
U can use an already ready to use container

`src/services/_root-container.ts`